scripts/gis-material-update/automation/Dockerfile

```
@@ -0,0 +1,35 @@
+FROM python:3.10
+
+ENV TORMAYS_DEPLOYMENT_PROFILE=docker_development
+RUN apt-get update
+
+RUN apt-get install -y software-properties-common wget gdal-bin libgdal-dev gettext-base python3-launchpadlib
+RUN add-apt-repository ppa:rmescandon/yq
+RUN apt install -y yq
```

The best practices say that:
1. Don't install recommended packages.
2. Clean cache after installing, to remove unnecessary files from the built image.
3. Add the dependencies one per row in alphabetical order. This makes it easier to track changes to them.
4. Don't mix apt-get and apt, it's just confusing.
```suggestion
RUN apt-get update  \
    && apt-get install --no-install-recommends -y \
    gdal-bin \
    gettext-base \
    libgdal-dev \
    python3-launchpadlib  \
    software-properties-common \
    wget \
    && apt-get clean
RUN add-apt-repository ppa:rmescandon/yq
RUN apt-get install --no-install-recommends -y yq && apt-get clean
```

---

Change done.